src/util/Constants.js

```diff
@@ -14,68 +14,50 @@ exports.WSCodes = {
   4014: 'DISALLOWED_INTENTS',
 };
 
-const AllowedImageFormats = ['webp', 'png', 'jpg', 'jpeg', 'gif'];
+const AllowedImageFormats = ['webp', 'png', 'jpg', 'jpeg'];
 
 const AllowedImageSizes = [16, 32, 56, 64, 96, 128, 256, 300, 512, 600, 1024, 2048, 4096];
 
-function makeImageUrl(root, { format = 'webp', size } = {}) {
+function makeImageUrl(root, { hash, format = 'webp', forceStatic = false, size } = {}) {
   if (!['undefined', 'number'].includes(typeof size)) throw new TypeError('INVALID_TYPE', 'size', 'number');
-  if (format && !AllowedImageFormats.includes(format)) throw new Error('IMAGE_FORMAT', format);
+  if (!AllowedImageFormats.includes(format)) throw new Error('IMAGE_FORMAT', format);
   if (size && !AllowedImageSizes.includes(size)) throw new RangeError('IMAGE_SIZE', size);
-  return `${root}.${format}${size ? `?size=${size}` : ''}`;
+  if (forceStatic) format ??= 'webp';
+  else if (hash?.startsWith('a_')) format = 'gif';
```

Comment:
`format` cannot be `null` or `undefined` at this point, also not to repeat the default twice
```suggestion
  if (!forceStatic && hash?.startsWith('a_')) format = 'gif';
```

Reply:
As discord api don't provide something like static or dynamic so why don't use format only and remove the `forceStatic` if anyone wants it in static form they can pass `png` or `webp` in format.

Reply:
Which is why it's not resolved. (: